test/repository_test.rb

```diff
@@ -98,6 +106,10 @@
             UserRepository.create(user1),
             UserRepository.create(user2)
           ]
+
+          @created_at = DateTime.new(2015,4,1)
+          @phone_without_created_at = Phone.new(name: 'Iphone 5S', publisher: 'Apple')
+          @phone_with_created_at = Phone.new(name: 'Galaxy s5', publisher: 'Samsung', created_at: @created_at)
```

Comment:
this fixture is wrong, it must be setup with `PhoneRepository.persist` so `id` is assigned.